server/test/tokensIntegrationTests.js

```diff
@@ -0,0 +1,114 @@
+const chai = require('chai');
+const assert = chai.assert;
+const chaiHttp = require('chai-http');
+const sinon = require('sinon');
+const app = require('../app');
+const db = require('../lib/db');
+
+chai.use(chaiHttp);
+chai.should();
+
+describe("Token Integration Tests: ", () => {
+    describe("Create an Account - POST & GET", () => {
+        var organization = "New Test Organization";
+        var token = "";
+
+        before(() => {
+
+        });
+
+        beforeEach(() => {
+
+        });
+
+        it("POST token/", (done) => {
+            chai.request(app)
+                .post('/api/tokens/')
+                .send({organization: organization})
+                .end((err, res) => {
+                    res.should.have.status(201);
+                    token = res.body.token;
+                    done();
+                });
+        });
+
+        it("GET token/:token", (done) => {
+            chai.request(app)
+                .get(`/api/tokens/${token}`)
+                .end((err, res) => {
+                    res.should.have.status(200);
+                    assert.equal(res.body.organization, organization);
+                    done();
+                });
+        });
+
+        afterEach(() => {
+
+        });
+
+        after(() =>{
+            db.executeQuery(`DELETE FROM tokens`).then((results) => {
+                done();
+            });
+        });
+    });
+
+    describe("Delete Account - DELETE & GET", () => {
+        var token = "a61c2fa0-e977-4982-9871-071514b2bc92";
+        var account_count = 0;
+
+        before(() => {
```

Comment:
I think you need to add done to the parameter: (done) => {..

Reply:
what line are you talking about?

Reply:
to line 60